_maps/RandomRuins/SpaceRuins/transit_booth.dmm

```diff
@@ -0,0 +1,657 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/template_noop,
+/area/template_noop)
+"b" = (
+/obj/effect/turf_decal/siding/dark{
+	dir = 8
+	},
+/obj/effect/turf_decal/siding/dark{
+	dir = 4
+	},
+/turf/open/floor/iron,
+/area/ruin/space/has_grav)
+"c" = (
+/obj/machinery/light/small/directional/north,
+/obj/structure/sign/warning/vacuum/external/directional/north,
+/turf/open/floor/plating,
+/area/ruin/space/has_grav)
+"d" = (
+/obj/effect/turf_decal/siding/dark{
+	dir = 4
+	},
+/obj/machinery/light/directional/east,
+/obj/effect/turf_decal/siding/dark{
+	dir = 8
+	},
+/turf/open/floor/iron,
+/area/ruin/space/has_grav)
+"e" = (
+/obj/machinery/vending/coffee,
+/turf/open/floor/iron/dark,
+/area/ruin/space/has_grav)
+"f" = (
+/obj/structure/window/reinforced{
+	dir = 1
+	},
+/obj/machinery/mass_driver{
+	id = "south"
+	},
+/obj/machinery/computer/pod/old/mass_driver_controller{
+	pixel_y = -5;
+	pixel_x = 25;
+	id = "south";
+	time = 0
+	},
+/turf/open/floor/plating,
+/area/ruin/space/has_grav)
+"g" = (
+/obj/machinery/door/airlock/external{
+	name = "Transit Booth Airlock"
+	},
+/obj/effect/mapping_helpers/airlock/cyclelink_helper{
+	dir = 8
+	},
+/turf/open/floor/plating,
+/area/ruin/space/has_grav)
+"h" = (
+/obj/effect/turf_decal/stripes/red/line,
+/obj/machinery/door/window/right/directional/west{
+	name = "Eastbound"
+	},
+/turf/open/floor/iron/dark,
+/area/ruin/space/has_grav{
+	name = "transitbooth"
+	})
```

Comment:
this is your issue. NEVER varedit areas

Reply:
Okay, that makes sense. What should I do to create a unique area name?

Reply:
Create an area subtype and set its name variable